lib/simplecov-material/version.rb

```diff
@@ -3,7 +3,7 @@
 module SimpleCov
   module Formatter
     class MaterialFormatter
-      VERSION = "0.4.0"
+      VERSION = "0.4.1"
```

Comment:
If you can remove this please apart of my publish cmd it does the version update automatically

Reply:
no prob.  done, pending push

Reply:
thanks!